sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -2473,175 +2479,17 @@
-                    "logic": [],
-                    "attributes": {},
-                    "overlay": {
-                        "style": "",
-                        "page": "",
-                        "left": "",
-                        "top": "",
-                        "width": "",
-                        "height": ""
-                    },
-                    "type": "htmlelement",
-                    "input": false,
-                    "tableView": false,
-                    "placeholder": "",
-                    "prefix": "",
-                    "suffix": "",
-                    "multiple": false,
-                    "defaultValue": null,
-                    "protected": false,
-                    "unique": false,
-                    "persistent": false,
-                    "clearOnHide": true,
-                    "refreshOn": "",
-                    "redrawOn": "",
-                    "dataGridLabel": false,
-                    "labelPosition": "top",
-                    "description": "",
-                    "errorLabel": "",
-                    "tooltip": "",
-                    "hideLabel": false,
-                    "tabindex": "",
-                    "disabled": false,
-                    "autofocus": false,
-                    "dbIndex": false,
-                    "customDefaultValue": "",
-                    "calculateValue": "",
-                    "calculateServer": false,
-                    "widget": null,
-                    "validateOn": "change",
-                    "validate": {
-                        "required": false,
-                        "custom": "",
-                        "customPrivate": false,
-                        "strictDateValidation": false,
-                        "multiple": false,
-                        "unique": false
-                    },
-                    "allowCalculateOverride": false,
-                    "encrypted": false,
-                    "showCharCount": false,
-                    "showWordCount": false,
-                    "allowMultipleMasks": false,
-                    "id": "e8kmoz"
-                },
                 {
                     "label": "How will you be attending the program",
                     "labelPosition": "top",
                     "widget": "choicesjs",
                     "placeholder": "Select how will you be attending the program",
                     "description": "",
-                    "tooltip": "",
+                    "tooltip": "A part-time program has a course load between 20 and 59%. A full-time program must have a course load of 60% or greater or\r\n40 and 60% for students with a permanent disability. If you are unsure, contact your school’s financial aid officer.",

```

Comment:
Please check the special character after "59%.".

Reply:
There was a junk character when I copied the content from Zenhub. It has been removed now.